### ISSUE TITLE
Make it easier to run same Command onSuccess and onFailure

### ISSUE THREAD

**Issue by ackvf** [gilamran/tsc-watch#76]
I was thinking about:

- either introduce third option, which would ignore both `onSuccess` and `onFailure`
- or make `--onSuccess` `--ignoreErrors`
- or do it similar to try catch finally, which would run after `onSuccess` or `onFailure` (if they are present)

**Comment by gilamran** [comment]
can you please provide examples? also why would you want that?

**Comment by ackvf** [comment]
There are use cases to run same scripts on both success and failure https://github.com/gilamran/tsc-watch/issues/31

So basically, I want this. _(Notice, I later moved the commands to `start` script so that the `dev` command is not cluttered.)_
```
"dev": "tsc-watch --onSuccess \"npm start\" --onFailure \"npm start\"",
```

And these would be the example usages from scripts in package.json for each of the proposed methods:

```
// 1
"dev": "tsc-watch --onFinished \"npm start\"",

// 2
"dev": "tsc-watch --onSuccess \"npm start\" --ignoreErrors",

// 3
"dev": "tsc-watch --onSuccess \"npm run ifSucceeded\" --onFailure \"npm run ifFailed\" --finally \"npm run performCleaningForExample\"",
// instead of
"dev": "tsc-watch --onSuccess \"npm run ifSucceeded && performCleaningForExample\" --onFailure \"npm run ifFailed && performCleaningForExample\"",
```

**Comment by gilamran** [comment]
Done.
Added `onCompilationComplete` option.
Available on version 4.1.1

**Comment by ackvf** [comment]
Amazing!, thank you!
